Description: install fsousage lib into private path
 The library is only used by fsousaged plugins. In Debian
 it's installed to a private location, since we do not keep
 track of the API compability.
 This patch also automatically adds RPATH to the plugins.
Author: Sebastian Reichel <sre@debian.org>
--- a/src/lib/Makefile.am
+++ b/src/lib/Makefile.am
@@ -2,6 +2,8 @@
 
 NULL =
 
+libdir = @libdir@/cornucopia/libs/fsousage
+
 AM_CPPFLAGS = \
 	-I$(top_srcdir)/src/lib \
 	$(FSO_CFLAGS) \
